VideoPlusChat/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="purple_200">#FFBB86FC</color>
```

Comment:
This is what I meant in a previous comment, you can access to the resources from this file without the need to create a kotlin wrapper :-)